xml/System.IdentityModel.Tokens/Saml2AuthenticationContext.xml

```diff
@@ -177,4 +177,4 @@
       </Docs>
     </Member>
   </Members>
-</Type>
+</Type>
```

Comment:
Do you mean using the reference source to get to these internal classes/members?